### ISSUE TITLE
Add a trigger CDC mode

### ISSUE THREAD

**Issue by hariso** [ConduitIO/conduit-connector-postgres#118]
### Feature description

Currently we have a CDC mode which uses PostgreSQL's WAL. We'd also like to have a CDC mode which uses PostgreSQL's triggers.

**Comment by rauanmayemir** [comment]
Does trigger mean calling `pg_notify` on each op?

**Comment by hariso** [comment]
@rauanmayemir Not really, I meant this: https://www.postgresql.org/docs/current/sql-createtrigger.html. IIRC, the motivation was to provide a way to get the changes in case WAL isn't available. I see a PR was merged ([link](https://github.com/ConduitIO/conduit-connector-postgres/pull/79)), so I need to check if there's some outstanding work left here.
